app/src/main/java/com/xa/xpensauditor/SetLimitActivity.java

```diff
@@ -148,14 +148,14 @@ public void onClick(View view) {
             public void onDataChange(@NonNull DataSnapshot snapshot) {
                 for (DataSnapshot snapshot3 : snapshot.getChildren()) {
                     if (Integer.parseInt(snapshot3.child("Day").getValue().toString()) == (thisDay)) {
-                        String amm = (snapshot3.child("Amount").getValue()).toString();
+                        String amm = (snapshot3.child("Amount").getValue().toString());
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' child has incorrect indentation level 24, expected level should be one of the following: 16, 18, 20.